data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -101,7 +86,7 @@ public static void main(String[] args) throws IOException, ParseException,
 
 		// generate Dot
 //		lineageMain.generateLineageDotFromDB();
-		lineageMain.generateLineageDot();
+		//lineageMain.generateLineageDot();
 
 		System.out.println("End of program");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS106)